documentation/src/main/asciidoc/topics/hotrod_protocol.adoc

```diff
@@ -2034,13 +2034,26 @@ Here are the current values for each flag: +
 .Infinispan versions
 TIP: This version of the protocol is implemented since Infinispan 9.2
 
-This Hot Rod protocol version adds support for transaction operations.
-It includes 3 new operations:
+This Hot Rod protocol version adds support for transaction operations and multimap caches.
+
+.New operations for transactions:
 
 * Prepare, with the transaction write set (i.e. modified keys), it tries to prepare and validate the transaction in the server.
 * Commit, commits a prepared transaction.
 * Rollback, rollbacks a prepared transaction.
 
+. New operations to manipulate multimap caches.
```

Comment:
any reason why this is in the middle of the transactions section?

Reply:
I moved it above